arch/xtensa/soc/esp32/Kconfig.defconfig

```diff
@@ -0,0 +1,15 @@
+# Kconfig - ESP32 board configuration
+#
+# Copyright (c) 2017 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_ESP32
+
+config SOC
+	string
+	default esp32
+
+config IRQ_OFFLOAD_INTNUM
+	default 7
```

Comment:
IRQ offload isn't working yet.  I've tried with two IRQ nos. that are available for software interrupts, and all I get is an exception.  There are more pressing matters at the moment but this is something that have to be fixed eventually.